### DISCUSSION TITLE
pc部署alist服务以后，在同局域网的其他设备上无法访问（之前是成功的，）

### DISCUSSION THREAD

**Discussion by kuuiom** [AlistGo/alist#7504] [General]
在pc上无论是使用127.0.0.1:5244还是本机ip：5244，都可以成功进入访问界面。如图
![pic2](https://github.com/user-attachments/assets/fa7f4a97-e2ef-4e4a-affd-b0a8068d6853)
但是在同局域网下的其他设备无法访问，请问这是为什么
![a5422780b550656b7d7e7275baf6a28](https://github.com/user-attachments/assets/47ce8d18-f776-4e85-b670-5b8d345d372f)

**Comment by sangchengfang** [comment]
可能是防火墙问题
